lib/phoenix/endpoint/attempt_code_reload_plug.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Phoenix.Endpoint.AttemptCodeReloadPlug do
+  @moduledoc ~S"""
```

Comment:
Please keep this as `@moduledoc false`, as it should be private, but keep those notes as code comments!

Reply:
Ah, you would need to use this from Bandit? Which then means it has to be public?

Reply:
Yup!